tests/test_compute_utils.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+
+import desc.io
+from desc.grid import ConcentricGrid, LinearGrid
+from desc.compute.utils import (
+    _get_grid_surface,
+    compress,
+    surface_averages,
+    surface_integrals,
+)
+
+
+def random_grid(linear=False):
```

Comment:
I don't love the idea of having tests with random grid parameters. I'm worried that it could lead to cases where tests pass sometimes but not other times depending on the random conditions.

Reply:
I agree with Daniel, if the point is to make sure these work with a grid with non-uniform spacing or non-uniform resolution, I'd prefer if the grid generating function you use here just return set defaults for L,M,N